src/types/index.ts

```diff
@@ -76,18 +75,18 @@ export type DayzedProps = {
   minDate?: Date;
   monthsToDisplay: number;
   offset: number;
-  onDateSelected: (props: any) => void;
+  onDateSelected: (dateObj: any, event: React.SyntheticEvent) => void;
```

Comment:
shouldn't this be the event as the first parameter?

Reply:
no, this is the type for the `dayzed` library, where the event is the second parameter...